### PR TITLE
TASK-1036: Fixes media & phenotype viewers by pointing to new dynamic service

### PR DESCRIPTION
**Pull request by JamesJeffryes**
@briehl I've tested this code locally and it works against CI with the CI service wizard url. This version hard coded to the prod service wizard.

### CODE REVIEW

File: kbase-extension/static/kbase/js/widgets/function_output/modeling/KBModeling.js [hunk 1]
```diff
@@ -8,36 +8,52 @@ function KBModeling(token) {
...
+            var prom = $.ajax({
+                url: _url,
+                type: 'POST',
+                processData: false,
+                data: JSON.stringify(rpc),
+                beforeSend: function (xhr) {
+                    if (self.token)
+                        xhr.setRequestHeader("Authorization", self.token);
+                }
+            }).then(_callback)
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/function_output/modeling/KBModeling.js]: Missing semicolons on this line and below.

File: kbase-extension/static/kbase/js/widgets/function_output/modeling/kbaseTabTable.js [hunk 2]
```diff
@@ -393,21 +393,21 @@ return KBWidget({
 
         this.getBiochemReaction = function(id) {
         	var input = {reactions: [id]};
-            return self.kbapi('fba', 'get_reactions', {reactions: [id]})
+            return self.kbapi('biochem', 'get_reactions', {reactions: [id]})
```
Comment (briehl) [inline kbase-extension/static/kbase/js/widgets/function_output/modeling/kbaseTabTable.js]: missing semicolon

Comment (briehl) [reply kbase-extension/static/kbase/js/widgets/function_output/modeling/kbaseTabTable.js]: same with other additions.

**Comment by JamesJeffryes** [comment]
I'm sympathetic with your stomachache but hardcoded urls are better than perma-spinners for the RSV.

**Comment by kkellerlbl** [comment]
@briehl yeah, it's just another bandaid.  We can order some antacid for all of us.  :)

**Comment by briehl** [comment]
Ok. I'll merge and get this up on CI.

**Pull request merged**